test/test.js

```diff
@@ -2751,6 +2753,23 @@ describe('Page', function() {
     });
   });
 
+  describe('Page.setCacheEnabled', function() {
+    it('should enable or disable the cache based on the state passed', async({page}) => {
+      let responses = [];
+      page.on('response', response => {
+        if (response.url.includes('.js'))
+          responses.push(response);
+      });
+      await page.setCacheEnabled(false);
+      await page.goto('https://www.chromestatus.com/features', {waitUntil: 'networkidle2'});
```

Comment:
Can you please use one of the `test/assets/cached/` assets intead of `chromestatus.com` website? We aim for the hermetic tests.